configs/application/components.json

```diff
@@ -6,8 +6,12 @@
 				"url": "$applicationRoot/components/welcome/welcome.html",
 				"affinity": "workspaceComponents",
 				"frame": false,
-				"resizable": true,
-				"autoShow": true,
+				"options": {
+					"resizable": true,
+					"autoShow": true,
+					"backgroundColor": "#22262F",
```

Comment:
@kriswest Why did you add a background color?

Reply:
To stimulate discussion of what we should actually be putting here. We've had dead options in place for over a year and at present have no docs on what options are available. This is one that has been requested in the past - although it doesn't actually prevent a white flash as the component loads annoyingly. 

I'll be following up on the ticket shortly as we should be discussing which options to surface in this example (which is likely to be one of the prime ways developers know about options, whether or not we document them).

Reply:
As this doesn't actually prevent that white flash, I think it can be dropped. 

Is there anything else (in finsemble\src\common\FinsembleWindowData.d.ts) that you think would be useful to add into an explicit example @NinjaManatee?